### PR TITLE
Remove UTC usage on backend

### PR DESCRIPTION
**Pull request by ge0ffrey**

### CODE REVIEW

**Comment by ge0ffrey** [comment]
@Christopher-Chianelli I 've added the system property directly instead of the todo :)

File: optashift-employee-rostering-server/src/main/java/org/optaplanner/openshift/employeerostering/server/admin/AdminRestServiceImpl.java [hunk 1]
```diff
@@ -31,12 +32,14 @@
     @Override
     @Transactional
     public void resetApplication() {
+        // TODO receive zoneId as a parameter instead
+        ZoneId zoneId = ZoneId.systemDefault();
```
Comment (Christopher-Chianelli) [inline optashift-employee-rostering-server/src/main/java/org/optaplanner/openshift/employeerostering/server/admin/AdminRestServiceImpl.java]: Should be `SomeConfigClass.determineZoneId()`

File: optashift-employee-rostering-server/src/main/java/org/optaplanner/openshift/employeerostering/server/roster/RosterGenerator.java [hunk 2]
```diff
@@ -303,34 +309,53 @@ public RosterGenerator(EntityManager entityManager) {
 
     @PostConstruct
     public void setUpGeneratedData() {
+        ZoneId zoneId = determineZoneId();
+        setUpGeneratedData(zoneId);
+    }
+
+    private ZoneId determineZoneId() {
```
Comment (Christopher-Chianelli) [inline optashift-employee-rostering-server/src/main/java/org/optaplanner/openshift/employeerostering/server/roster/RosterGenerator.java]: Move to a config class and make it a static method

**Comment by ge0ffrey** [comment]
@Christopher-Chianelli fixes done, merging

**Pull request merged**